src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.NonWindows.nuspec

```diff
@@ -36,48 +36,42 @@
 
     <!-- netstandard2.0 -->
     <file src="netstandard2.0\build\common\MSTest.TestAdapter.props" target="build\netstandard2.0\" />
-    <file src="netstandard2.0\build\netfx-netcore-netstandard\MSTest.TestAdapter.targets" target="build\netstandard2.0\" />
-    <file src="netstandard2.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\netstandard2.0\" />
+    <file src="netstandard2.0\build\common\MSTest.TestAdapter.targets" target="build\netstandard2.0\" />
     <file src="netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll" target="build\netstandard2.0\" />
     <file src="netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.dll" target="build\netstandard2.0\" />
     <file src="netstandard2.0\Microsoft.TestPlatform.AdapterUtilities.dll" target="build\netstandard2.0\" />
 
     <!-- netcoreapp3.1 -->
     <file src="netcoreapp3.1\build\common\MSTest.TestAdapter.props" target="build\netcoreapp3.1\" />
-    <file src="netcoreapp3.1\build\netfx-netcore-netstandard\MSTest.TestAdapter.targets" target="build\netcoreapp3.1\" />
-    <file src="netcoreapp3.1\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\netcoreapp3.1\" />
+    <file src="netcoreapp3.1\build\common\MSTest.TestAdapter.targets" target="build\netcoreapp3.1\" />
```

Comment:
Is there any plan to remove netcore3.1 support?

Reply:
In v4 because it's a breaking change.